File: modul_6/PRAK605-2510817220008-NamiraHumaira.py
ordo = int(input())
print("Matriks A")
matriks_A = []
for i in range(ordo):
    angka_1 = list(map(int,input().split()))
    matriks_A.append(angka_1)
print("Matriks B")
matriks_B = []
for i in range(ordo):
    angka_2 = list(map(int,input().split()))
    matriks_B.append(angka_2)
matriks_AxB = [[0] * ordo for _ in range(ordo)]
for i in range(ordo):
    for j in range(ordo):
        for n in range(ordo):
            matriks_AxB[i][j] += matriks_A[i][n] * matriks_B[n][j]
print("\nMatriks AxB")
for angka_1 in matriks_AxB:
    print(" ".join(map(str,angka_1)))

File: modul_6/PRAK605-2510817220008-NamiraHumaira.c
#include <stdio.h>
int main()
{
    int ordo;
    scanf("%d", &ordo);
    printf("Matriks A \n");
    int matrixA[ordo][ordo], matrixB[ordo][ordo];
    for (int i = 0; i < ordo; i++)
    {
        for (int j = 0; j < ordo; j++)
        {
            scanf("%d", &matrixA[i][j]);
        }
    }
    printf("Matriks B \n");
    for (int i = 0; i < ordo; i++)
    {
        for (int j = 0; j < ordo; j++) 
        {
            scanf("%d", &matrixB[i][j]);
        }
    }
    int matrixAB[ordo][ordo];
    for (int i = 0; i < ordo; i++)
    {
        for (int j = 0; j < ordo; j++) 
        {
            matrixAB[i][j] = 0;
            for (int k = 0; k < ordo; k++) 
            {
                matrixAB[i][j] += matrixA[i][k] * matrixB[k][j];
            }
        }
    }
    printf("\nMatriks AXB\n");
    for (int i = 0; i < ordo; i++)
    {
        for (int j = 0; j < ordo; j++) 
        {
            printf("%d ", matrixAB[i][j]);
        }
        printf("\n");
    } 
    return 0;
}